etc/config/fsharp.amazon.properties

```diff
@@ -1,10 +1,10 @@
 compilers=&fsharp
-supportsBinary=false
+supportsBinary=true
 needsMulti=false
 compilerType=fsharp
 defaultCompiler=dotnet700fsharp
 
-group.fsharp.compilers=dotnet6011fsharp:dotnet700fsharp
+group.fsharp.compilers=dotnet6011fsharp:dotnet700fsharp:dotnettrunkfsharp
```

Comment:
we can adapt order of gcc, clang and go in the ui: `dotnettrunkfsharp;dotnet700fsharp;dotnet700fsharp` (same for c# and vb)

makes sense because someone experimenting on godbolt.org will likely want to test the latest version first (lts version is not important when it comes to godbolt).